.vsts-ci.yml

```diff
@@ -1,26 +1,48 @@
+resources:
+  repositories:
+  # shared library repository
+  - repository: arcade
+    type: github
+    endpoint: jofortes-personal
+    name: dotnet/arcade
+    ref: refs/heads/master
+
+variables:
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+
 phases:
-- phase: First_phase
-  queue:
-    name: Hosted VS2017
-  steps:
-  - script: echo Hello World!
+- template: /eng/common/templates/phases/base.yml@arcade
+  parameters:
+    queue:
+      name: dotnet-external-temp
+      timeoutInMinutes: 180
+      parallel: 3
+      matrix:
+        x64_checked_Innerloop:
+          buildConfig: checked
+          archType: x64
+    agentOs: Windows_NT
+    phaseName: Windows_NT
+    steps:
+    - task: DotNetCoreInstaller@0
```

Comment:
This PR is using `build.cmd`, which will in turn use dotnet CLI. Do we need an installer task?